.github/workflows/downsize-images.yml

```diff
@@ -0,0 +1,67 @@
+name: downsize-images-scheduled
+
+on:
+  pull_request: []
+  push:
+    branches:
+      - master
+
+jobs:
+  DownsizeImages:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Install imageMagick
+      run: |
+          echo "Installing ImageMagick"
+          wget https://imagemagick.org/download/binaries/magick
+          chmod +x magick         
+          sudo mv magick /usr/local/bin
+          which magick
```

Comment:
I am using `convert` the script but I can also change it to `magick` if that makes the installation easier

Reply:
Yes that would be good.

Reply:
alright, done with ab739fb6c4c2f878e7eecfcd1100921aada15e28. It was actually quite simple, I just had to change `convert` to `magick`...